onnxruntime/core/providers/js/js_execution_provider.cc

```diff
@@ -279,6 +279,10 @@ class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomai
 class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 11, 12, Gather);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 13, Gather);
 
+class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 1, 10, GatherElements);
```

Comment:
GatherElements for opset 1-10 is not documented. Is it necessary add it?

Reply:
That's true. GatherElements was introduced in opset-11 AFAIK. Will remove this registration. It will save the kernel registry from bloating.